booster/tools/booster/Server.hs

```diff
@@ -379,7 +382,6 @@ logLevelToKoreLogEntryMap :: Map.Map LogLevel [Text.Text]
 logLevelToKoreLogEntryMap =
     Map.fromList
         [ (LevelOther "SimplifyKore", ["DebugAttemptEquation", "DebugTerm"])
-        , (LevelOther "SimplifyJson", ["DebugAttemptEquation", "DebugTerm"])
```

Comment:
we still need to pass these (e.g. `DebugAttemptEquation`) to actually get context logging in the old backend, because it has multiple layers of filtering. We could just enable all of them tho...